Simplenote/src/main/java/com/automattic/simplenote/authentication/SignInFragment.kt

```diff
@@ -1,49 +1,137 @@
 package com.automattic.simplenote.authentication
 
+import android.app.Activity
+import android.content.Context
 import android.content.Intent
+import android.net.Uri
 import android.os.Bundle
+import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Button
 import android.widget.EditText
-import android.widget.TextView
+import android.widget.Toast
+import androidx.activity.result.contract.ActivityResultContracts
 import androidx.fragment.app.viewModels
 import com.automattic.simplenote.R
+import com.automattic.simplenote.Simplenote
 import com.automattic.simplenote.analytics.AnalyticsTracker
 import com.automattic.simplenote.analytics.AnalyticsTracker.Stat
 import com.automattic.simplenote.utils.NetworkUtils
+import com.automattic.simplenote.utils.StrUtils
+import com.automattic.simplenote.utils.WordPressUtils
 import com.automattic.simplenote.viewmodels.MagicLinkRequestUiState
 import com.automattic.simplenote.viewmodels.RequestMagicLinkViewModel
 import dagger.hilt.android.AndroidEntryPoint
+import net.openid.appauth.AuthorizationException
+import net.openid.appauth.AuthorizationResponse
+import net.openid.appauth.AuthorizationService
+import java.util.UUID
+
+const val TAG = "SignInFragment"
 
 @AndroidEntryPoint
 class SignInFragment: MagicLinkableFragment() {
 
     val viewModel: RequestMagicLinkViewModel by viewModels()
 
-    private var loginWithPassword: TextView? = null
+
+    private var loginWithWordpress: Button? = null
```

Comment:
Looks like we can remove loginWithWordpress from here, since it's only used to set a listener in inflateLayout.